libcodechecker/server/api/report_server.py

```diff
@@ -2047,18 +2038,27 @@ def massStoreRun(self, name, tag, version, b64zip, force):
 
                     LOG.debug("Storing done for report " + str(report_id))
 
-                if len(report_path_and_events):
-                    store_handler.changePathAndEvents(self.__storage_session,
-                                                      run_id,
-                                                      report_path_and_events)
+            reports_to_delete = set()
+            for bug_hash, reports in hash_map_reports.items():
+                if bug_hash in new_bug_hashes:
+                    reports_to_delete.update(map(lambda x: x.id, reports))
+                else:
+                    fix_time = datetime.now()
```

Comment:
The fix time has to be the same in a run storage. Use `run_history_time` instead of datetime.